### ISSUE TITLE
docker-compose 1.27.4 silent ignore "configs"

### ISSUE THREAD

**Issue by hholst80** [docker/compose#8016]
Docker-compose 1.27.4 silently drops the "configs" section when deploying on a non-swarm.

```
version: '3.8'

services:
  haproxy:
    image: haproxy:2.2
    configs:
      - source: haproxy.cfg
        target: /usr/local/etc/haproxy/haproxy.cfg

configs:
  haproxy.cfg:
    file: ./haproxy/haproxy.cfg
```

**Comment by hholst80** [comment]
Related issue https://github.com/docker/compose/issues/5110

**Comment by hholst80** [comment]
Fixed in 1.28.0

```
WARNING: Some services (haproxy) use the 'configs' key, which will be ignored. Compose does not support 'configs' configuration - use `docker stack deploy` to deploy to a swarm.
```
